rskj-core/src/test/java/co/rsk/peg/federation/VoteFederationChangeTest.java

```diff
@@ -330,6 +343,25 @@ void voteCommitFederation_commit1MemberFederation_returnsInsufficientMembersResp
         assertEquals(FederationChangeResponseCode.INSUFFICIENT_MEMBERS.getCode(), firstVoteResult);
     }
 
+    @Test
+    void voteCommitFederation_withMismatchedHashes_returnsFederationNonExistentResponseCode() {
```

Comment:
The return in this case is `PENDING_FEDERATION_MISMATCHED_HASH`.

Reply:
you right! Thanks

Reply:
updated